src/compute/src/compute_state.rs

```diff
@@ -294,6 +299,21 @@ impl ComputeState {
         // Remember the maintenance interval locally to avoid reading it from the config set on
         // every server iteration.
         self.server_maintenance_interval = COMPUTE_SERVER_MAINTENANCE_INTERVAL.get(config);
+
+        if self.replica_expiration.is_none() {
+            let offset = COMPUTE_REPLICA_EXPIRATION.get(&self.worker_config);
+            if !offset.is_zero() {
+                // TODO(sdht0): consistently send now() from environmentd.
+                let now = mz_ore::now::SYSTEM_TIME.clone()();
+                let offset: EpochMillis = offset
+                    .as_millis()
+                    .try_into()
+                    .expect("Duration did not fit within u64");
+                let replica_expiration = Timestamp::new(now + offset);
+                info!("Replica frontier expires at {:?}", replica_expiration);
```

Comment:
```suggestion
                info!("replica expires at time {replica_expiration:?}");
```

I'm not sure what a "replica frontier" is. Also, check out the [style guide on log messages](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#log-message-style), which suggests to not use capitalization at the beginning.

Reply:
Done!